<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PNG2GIFsGUI</class>
 <widget class="QDialog" name="PNG2GIFsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>228</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PNG to GIF+Mask pair converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="_resources/png2gifs.qrc">
    <normaloff>:/cat_png2gif.ico</normaloff>:/cat_png2gif.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Target path:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="targetPath">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is a path where will be generated your GIF files.&lt;/p&gt;&lt;p&gt;If you wasn't define it, all files will be generated into same folder where placed original files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is a path where will be generated your GIF files.&lt;/p&gt;&lt;p&gt;If you wasn't define it, all files will be generated into same folder where placed original files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="targetPathBrowse">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drop here your PNG files and they are will be converted into SMBX format. Result will appear in the same folder. But if you was defined target path, all converted files will be saved in your defined path.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0); border-style: dashed; border-width: 2px; border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="LabelDropFiles">
        <property name="styleSheet">
         <string notr="true">border-width: 0px;</string>
        </property>
        <property name="text">
         <string>Drag and Drop your PNG files to get SMBX-compatible masked GIF files</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="_resources/png2gifs.qrc"/>
 </resources>
 <connections/>
</ui>
